src/Modules/CalcPerform.lua

```diff
@@ -810,6 +810,10 @@ local function doActorMisc(env, actor)
 			modDB:NewMod("SpellDodgeChance", "BASE", m_floor(15 * effect), "Elusive")
 			modDB:NewMod("MovementSpeed", "INC", m_floor(30 * effect), "Elusive")
 		end
+		if modDB:Flag(nil, "OnConsecratedGround") then
```

Comment:
Change this to use the Condition and you don't have to have both lines in ConfigOptions
```suggestion
		if modDB:Flag(nil, "Condition:OnConsecratedGround") then
```

Reply:
Done